server/index.js

```diff
@@ -18,6 +19,8 @@ const options = {
 app.get('/products', (req, res) => {
   axios.get(`${API_BASE_URL}/products`, options)
     .then((response) => {
+      console.log(response.data);
```

Comment:
Did you mean to leave this here?

Reply:
No, all the console log should be removed.